src/core/mac/mac.hpp

```diff
@@ -647,6 +647,22 @@ class Mac: public ThreadNetifLocator
      */
     bool RadioSupportsRetries(void);
 
+    /**
+     * This method returns the amount of time the radio has spent in Tx mode.
+     *
+     * @returns  A pointer to the total Tx time.
+     *
+     */
+    const uint32_t *GetRadioTxTotalTime(void) { return &mTxTotal; }
```

Comment:
Why return a pointer?  Would it not be more appropriate to just return a uint32_t?  Same for GetRadioRxTotalTime below.